app/views/layouts/_nav.html.erb

```diff
@@ -1,14 +1,14 @@
 <nav class='custom_nav <%= "landing" unless @current_page && @current_page == "new" %>'>
   <div class="nav-wrapper">
-    <%= "#{@event.event_template} darken-4 " if controller_name == "events" && action_name == "show"%>
+      <!-- "#{@event.event_template} darken-4" if controller_name == "events" && action_name == "show" -->
```

Comment:
If code is obsolete, it should be removed as opposed to just getting commented out.